webapp/sdf/src/components/list-component/list-component.vue

```diff
@@ -59,7 +68,12 @@ export default {
   computed: {
     sortedArray() {
       var newArray = [...this.itemList];
-      return newArray.sort((a, b) => b.votes - a.votes);
+      return newArray.sort((a, b) => {
+        if (this.orderAscending)
+          return b.votes - a.votes
+        else
+          return a.votes - b.votes
```

Comment:
```suggestion
        if (this.orderAscending) return b.votes - a.votes
        return a.votes - b.votes
```

That's a guard-clause :point_up: which is always easier to read than adding curly braces and nesting.